src/main/java/fi/otavanopisto/kuntaapi/server/persistence/dao/ArchivedIdentifierDAO.java

```diff
@@ -0,0 +1,203 @@
+    
+    return getSingleResult(entityManager.createQuery(criteria));
+  }
+//  
+//  
+//
+//  /**
+//   * Finds identifier by type, source and Kunta API id
+//   * 
+//   * @param type identifier type
+//   * @param source source
+//   * @param kuntaApiId Kunta API id 
+//   * @return found identifier or null if non found
+//   */
+//  public ArchivedIdentifier findByTypeSourceAndKuntaApiId(String type, String source, String kuntaApiId) {
+//    EntityManager entityManager = getEntityManager();
+//
+//    CriteriaBuilder criteriaBuilder = entityManager.getCriteriaBuilder();
+//    CriteriaQuery<ArchivedIdentifier> criteria = criteriaBuilder.createQuery(ArchivedIdentifier.class);
+//    Root<ArchivedIdentifier> root = criteria.from(ArchivedIdentifier.class);
+//    criteria.select(root);
+//    criteria.where(
+//      criteriaBuilder.and(
+//          criteriaBuilder.equal(root.get(ArchivedIdentifier_.type), type),
+//          criteriaBuilder.equal(root.get(ArchivedIdentifier_.source), source),
+//          criteriaBuilder.equal(root.get(ArchivedIdentifier_.kuntaApiId), kuntaApiId)          
+//      )
+//    );
+//    
+//    return getSingleResult(entityManager.createQuery(criteria));
+//  }
+//
+//  public List<ArchivedIdentifier> listBySourceAndType(String source, String type) {
+//    EntityManager entityManager = getEntityManager();
+//
+//    CriteriaBuilder criteriaBuilder = entityManager.getCriteriaBuilder();
+//    CriteriaQuery<ArchivedIdentifier> criteria = criteriaBuilder.createQuery(ArchivedIdentifier.class);
+//    Root<ArchivedIdentifier> root = criteria.from(ArchivedIdentifier.class);
+//    criteria.select(root);
+//    criteria.where(
+//      criteriaBuilder.and(
+//        criteriaBuilder.equal(root.get(ArchivedIdentifier_.type), type),
+//        criteriaBuilder.equal(root.get(ArchivedIdentifier_.source), source)
+//      )
+//    );
+//    
+//    return entityManager.createQuery(criteria).getResultList();
+//  }
+//
+//  public List<ArchivedIdentifier> listByOrganizationIdAndSourceAndType(String organizationKuntaApiId, String source, String type) {
+//    EntityManager entityManager = getEntityManager();
+//
+//    CriteriaBuilder criteriaBuilder = entityManager.getCriteriaBuilder();
+//    CriteriaQuery<ArchivedIdentifier> criteria = criteriaBuilder.createQuery(ArchivedIdentifier.class);
+//    Root<ArchivedIdentifier> root = criteria.from(ArchivedIdentifier.class);
+//    criteria.select(root);
+//    criteria.where(
+//      criteriaBuilder.and(
+//        criteriaBuilder.equal(root.get(ArchivedIdentifier_.organizationKuntaApiId), organizationKuntaApiId),
+//        criteriaBuilder.equal(root.get(ArchivedIdentifier_.type), type),
+//        criteriaBuilder.equal(root.get(ArchivedIdentifier_.source), source)
+//      )
+//    );
+//    
+//    return entityManager.createQuery(criteria).getResultList();
+//  }
+//
+//  public ArchivedIdentifier updateOrderIndex(ArchivedIdentifier identifier, Long orderIndex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)